Core/clim-basic/utils.lisp

```diff
@@ -743,3 +728,195 @@ a flag CLOSED is T then beginning and end of the list are consecutive too."
      ((and *print-pretty* *print-readably*)
       (simple-pprint-object ,sink ,self))
      (t ,@body)))
+
+
+
+(defun %line-breaks-1 (string width initial-offset margin start end)
```

Comment:
Add a comment explaining what this functions does.

Reply:
addressed in the newest commit